src/Service.Tests/Configuration/ConfigurationTests.cs

```diff
@@ -3499,10 +3499,68 @@ public void TestNonExistingLogLevels(Level expectedLevel)
             }
         }
 
+        /// <summary>
+        /// Tests different loglevel values to see if they are serialized correctly to the Json config
+        /// </summary>
+        [DataTestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        [DataRow(ExtendedLogLevel.Debug)]
+        [DataRow(ExtendedLogLevel.Warning)]
+        [DataRow(ExtendedLogLevel.None)]
+        [DataRow(null)]
+        public void LogLevelSerialization(ExtendedLogLevel expectedLevel)
+        {
+            RuntimeConfig configWithCustomLogLevel = InitializeRuntimeWithLogLevel(expectedLevel);
+            string configWithCustomLogLevelJson = configWithCustomLogLevel.ToJson();
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(configWithCustomLogLevelJson, out RuntimeConfig deserializedRuntimeConfig));
+
+            string serializedConfig = deserializedRuntimeConfig.ToJson();
+
+            using (JsonDocument parsedDocument = JsonDocument.Parse(serializedConfig))
+            {
+                JsonElement root = parsedDocument.RootElement;
+
+                //Validate log-level property exists in runtime
+                JsonElement runtimeElement = root.GetProperty("runtime");
+                bool logLevelPropertyExists = runtimeElement.TryGetProperty("log-level", out JsonElement logLevelElement);
+                Assert.AreEqual(expected: true, actual: logLevelPropertyExists);
+
+                //Validate level property inside log-level is of expected value
+                bool levelPropertyExists = logLevelElement.TryGetProperty("level", out JsonElement levelElement);
+                Assert.AreEqual(expected: true, actual: levelPropertyExists);
+                Assert.AreEqual(expectedLevel.ToString().ToLower(), levelElement.GetString());
+            }
+        }
+
+        /// <summary>
+        /// Tests serialization for the case that the loglevel property is not included,
+        /// to ensure that it is not written to the Json config.
+        /// </summary>
+        [TestMethod]
+        [TestCategory(TestCategory.MSSQL)]
+        public void LogLevelSerializationForNonExistentProperty()
+        {
+            RuntimeConfig runtimeConfig = CreateBasicRuntimeConfigWithNoEntity();
+            string configJson = runtimeConfig.ToJson();
+            Assert.IsTrue(RuntimeConfigLoader.TryParseConfig(configJson, out RuntimeConfig deserializedRuntimeConfig));
```

Comment:
what are you trying to confirm here? that `CreateBasicRuntimeConfigWithNoEntity();` returned a valid runtimeconfig?

Reply:
Deleted test as it was manually tested that there is no accidental writing made to the config file